packages/tools/src/apm.coffee.md

```diff
@@ -53,3 +53,27 @@ Install Atom packages with APM.
           if: install.length
         , (err) =>
           @log message: "APM Installed Packages: #{install.join ', '}"
+
+
+    module.exports.uninstall = ({options}, callback) ->
+    options.name = options.argument if options.argument?
+    options.name = [options.name] if typeof options.name is 'string'
+    installed = []
+    @system.execute
+      cmd: "apm list --installed --json #{global}"
+      code: [0, 1]
+      stdout_log: false
+      shy: true
+    , (err, {stdout}) ->
+      throw err if err
+      pkgs = JSON.parse stdout
+      pkgs = Object.keys pkgs.dependencies
+      installed = pkgs
+    @call ->
+      uninstall = options.name.filter (pkg) -> pkg in installed
+      @system.execute
+        if: uninstall.length
+        cmd: "apm uninstall #{uninstall.join ' '}"
+        sudo: options.sudo
+      , (err) =>
+        @log message: "apm uninstalled packages: #{install.join ', '}"
```

Comment:
Again, you were advise to follow the conventions from the `npm` action, any other actions would work as well.

Reply:
Updating with separated modules for install and uninstall and options, source, exports and dependency blocks